Added additional information to Returns Attributes page (#1707)
magento/merchdocs: src/stores/attributes-returns.md

1. In the left panel, choose **Manage Labels/Options**.

> 1. In the **Manage Titles (Size, Color, etc.)** section, enter the title value for each Store View.
> 
> {:.bs-callout-info}
> If the **Input Type** is `Dropdown` you can manage options in **Manage Options (Values of Your Attribute)** section.
> Click <span class="btn">Add Option</span> for adding new option, enter the title for Admin and each Store View.
> Click **Is Default** radiobutton, to make an option selected by default.
> Click **Delete** to delete an option.

Reviewer: This would be better as a nested bullet list for the step.

```suggestion
1. In the **Manage Titles (Size, Color, etc.)** section, enter the label for each store view.

1. If the **Input Type** for the attribute is `Dropdown`, manage the options in the **Manage Options (Values of Your Attribute)** section.

   - To add an option, click <span class="btn">Add Option</span> and enter the label for Admin and each store view.
   - To make an option selected by default, choose **Is Default**.
   - To remove an option, click **Delete**.
```

> 1. To save changes, click <span class="btn">Save Attribute</span>.

Reviewer: Let's replace the screen capture (replace the file using the same file name) with one that shows the options as described:

![image](https://user-images.githubusercontent.com/1128913/156060842-6395c6cc-a533-47d4-84ac-ef87a85deb7e.png)
